README.md

```diff
@@ -1,64 +1,88 @@
-<p align="center">
-<img src="https://github.com/CesiumGS/cesium/wiki/logos/Cesium_Logo_Color.jpg" width="50%" />
-</p>
+# CesiumJS
+
+![Cesium](https://github.com/CesiumGS/cesium/wiki/logos/Cesium_Logo_Color.jpg)
 
 [![Build Status](https://travis-ci.com/CesiumGS/cesium.svg?branch=main)](https://travis-ci.com/CesiumGS/cesium)
 [![npm](https://img.shields.io/npm/v/cesium)](https://www.npmjs.com/package/cesium)
 [![Docs](https://img.shields.io/badge/docs-online-orange.svg)](https://cesium.com/learn/)
 
 CesiumJS is a JavaScript library for creating 3D globes and 2D maps in a web browser without a plugin. It uses WebGL for hardware-accelerated graphics, and is cross-platform, cross-browser, and tuned for dynamic-data visualization.
 
-[CesiumJS Homepage](https://cesium.com/cesiumjs)
+Built on open formats, CesiumJS is designed for robust interoperability and scaling for massive datasets.
+
+[Examples](https://sandcastle.cesium.com/)
+:earth_africa: [Docs](https://cesium.com/learn/cesiumjs-learn/) :earth_asia: [Website](https://cesium.com/cesiumjs) :earth_americas: [Forum](https://community.cesium.com/)
+
+## :rocket: Get Started
 
-[CesiumJS Features Checklist](https://github.com/CesiumGS/cesium/wiki/CesiumJS-Features-Checklist)
+Visit the [Downloads page](https://cesium.com/downloads/) to download a pre-built copy of CesiumJS.
 
-### :rocket: Get Started
+### npm & yarn
 
-Visit the [Downloads page](https://cesium.com/downloads/) or install the [npm package](https://www.npmjs.com/package/cesium):
+If you’re building your application using a module bundler such as Webpack, Parcel, or Rollup, you can install CesiumJS via the [`cesium` npm package](https://www.npmjs.com/package/cesium):
 
 ```sh
-npm install cesium
+npm install cesium --save
+```
+
+Then, import CesiumJS in your app code:
+
+```js
+import * as Cesium from "cesium";
+import "cesium/Build/Cesium/Widgets/widgets.css";
+
+const viewer = new CesiumWidgets.Viewer("cesiumContainer");
+```
+
+Or, import individual modules to benefit from tree shaking optmtimizations through most build tools:
+
+```js
+import { Viewer } from "cesium";
+import "cesium/Build/Cesium/Widgets/widgets.css";
+
+const viewer = new Viewer("cesiumContainer");
 ```
 
-Have questions? Ask them on the [community forum](https://community.cesium.com/).
+#### Packages
+
+In addition to the `cesium` package, CesiumJS is also distrubuted as scoped npm packages for more specific dependency management:
+
+- [`@cesium/engine`](./packages/engine/README.md) - CesiumJS's core, rendering, and data APIs
+- [`@cesium/widgets`](./packages/widgets/README.md) - A widgets library for use with CesiumJS
+
+### CDN
```

Comment:
I personally prefer having the CDN link directly in the README, [as Babylon does](https://github.com/BabylonJS/Babylon.js), especially since right now we're already linking to the quick start guide right below this.

Reply:
I linked to the quickstart guide because we don't have an ""official"" static CDN as the URL changes release to release.